snc.sh

```diff
@@ -164,11 +159,37 @@ DYNAMIC_DATA=$(base64 -w0 node-sizing-enabled.env) envsubst < 99_master-node-siz
 export OPENSHIFT_INSTALL_INVOKER="codeReadyContainers"
```

Comment:
Good opportunity to change this?

Reply:
Sure we can change it but this is mostly on openshift telemetry side and this is what is reserved for us, have to check about it.